ArduCopter/defines.h

```diff
@@ -107,6 +107,13 @@ enum GuidedMode {
     Guided_Angle,
 };
 
+// Airmode
+enum class AirMode {
+    None,
```

Comment:
Our convention here is to use caps for the enum entries.  Not going to block this based on that - especially as two of the entries would cause compilation errors because of namespace pollution issues!

Reply:
It's an enum class so is this really true?

Reply:
The convention?  Yes.

The namespace-pollution?

```
In file included from ../../ArduCopter/Copter.h:70:0,
                 from ../../ArduCopter/mode_althold.cpp:1:
../../ArduCopter/defines.h:6:33: error: expected identifier before numeric constant
 #define ENABLED                 1
                                 ^
../../ArduCopter/AP_Arming.h:12:9: note: in expansion of macro 'ENABLED'
         ENABLED = 0,
         ^
compilation terminated due to -Wfatal-errors.
```

for:

```
diff --git a/ArduCopter/AP_Arming.h b/ArduCopter/AP_Arming.h
index 0a1a27e2e2..cc91de743e 100644
--- a/ArduCopter/AP_Arming.h
+++ b/ArduCopter/AP_Arming.h
@@ -8,6 +8,12 @@ public:
     friend class Copter;
     friend class ToyMode;
 
+    enum class Fred {
+        ENABLED = 0,
+        DISABLED = 1,
+        TERTIUS = 2,
+    };
+
     AP_Arming_Copter() : AP_Arming()
     {
         // default REQUIRE parameter to 1 (Copter does not have an
```

Reply:
Ah that kind of pollution - I stand corrected!

Reply:
Ok fixed!